src/main/java/com/auth0/json/mgmt/client/Credential.java

```diff
@@ -0,0 +1,129 @@
+package com.auth0.json.mgmt.client;
```

Comment:
Should a `Credential` have an `expiresAt` and `updatedAt`? https://github.com/auth0/api2/blob/master/api/credentials/schemas/credential_response.js#L41-L50

Reply:
Good catch. Pushed new commit with additional fields and tests.